crates/lace-bin/src/typechecking/static_pass/mod.rs

```diff
@@ -0,0 +1,11 @@
+use crate::parser::ast::Ast;
+use crate::typechecking::ctx::TypeContext;
+use crate::typechecking::static_pass::find_statics::find_statics_item;
+
+pub mod find_statics;
+
+pub fn find_statics_ast(ast: &Ast, global_ctx: &mut TypeContext) {
```

Comment:
What is this about? Also why is it split over 2 tiny files?

Reply:
well in the future this will be much bigger. All structs and traits need to be found here, as well as functions defined in the scopes of other functions. In the end this will be almost a full ast traversal.

Reply:
It's split by the same structure as the typechecking code, even though this is still much smaller.

Reply:
Sounds like that could be in the stack graph or something as well? What is it doing right now?